packages/web-react/src/components/Toast/ToastBar.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import React, { MutableRefObject, useRef } from 'react';
+import { Transition, TransitionStatus } from 'react-transition-group';
+import { useStyleProps } from '../../hooks';
+import { SpiritToastBarProps } from '../../types';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+import { VisuallyHidden } from '../VisuallyHidden';
+import { DEFAULT_ICONS, ICON_BOX_SIZE, TRANSITIONING_STYLES, TRANSITION_DURATION } from './constants';
+import { useToastBarStyleProps } from './useToastBarStyleProps';
+
+const ToastBar = (props: SpiritToastBarProps) => {
+  const {
+    id,
+    children,
+    closeLabel = 'Close',
+    color = 'inverted',
```

Comment:
Minor suggestion: if you declare the `inverted` type you can use it also as a value for this default.

Reply:
I moved this string to constants because I also use it in the `useToastIcon` hook.